prow/plank/plank_test.go

```diff
@@ -405,22 +405,25 @@ func TestSyncJenkinsJob(t *testing.T) {
 		}
 		c.setPending(tc.pendingJobs)
 
-		if err := c.syncJenkinsJob(tc.pj); err != nil != tc.expectedError {
+		reports := make(chan kube.ProwJob, 100)
```

Comment:
Don't want this to block if someone accidentally sends more than one, unclear if we want something more sophisticated than just making it "large enough"